tendrl-monitoring-integration.spec

```diff
@@ -54,7 +54,9 @@ rm -rf html/.{doctrees,buildinfo}
 install -m  0755  --directory $RPM_BUILD_ROOT%{_sysconfdir}/tendrl/monitoring-integration
 install -m  0755  --directory $RPM_BUILD_ROOT%{_sysconfdir}/tendrl/monitoring-integration/grafana
 install -m  0755  --directory $RPM_BUILD_ROOT%{_sysconfdir}/tendrl/monitoring-integration/grafana/dashboards
+install -m  0755 --directory $RPM_BUILD_ROOT%{_sysconfdir}/firewalld/services
 install -d %{buildroot}%{_localstatedir}/lib/grafana/plugins/
+install -Dm 0644 etc/firewalld/services/tendrl-monitoring-integration.xml $RPM_BUILD_ROOT%{_sysconfdir}/firewalld/services
```

Comment:
When packaging firewalld service file into `/usr/lib/firewalld/services/` (see below), it's not necessary to create this directory in `/etc`.

Reply:
Got it, I will move to /usr/lib/firewalld/services